rosdep/python.yaml

```diff
@@ -6306,9 +6306,7 @@ wxpython:
   opensuse: [python-wxGTK]
   rhel: [wxPython-devel]
   ubuntu:
-    artful: [python-wxgtk3.0]
-    bionic: [python-wxgtk3.0]
-    cosmic: [python-wxgtk3.0]
+    '*': [python-wxgtk3.0]
```

Comment:
I think we should change it to exactly:

```
ubuntu: [python-wxgtk3.0]
```

And remove everything else.  We don't need to keep the keys for unsupported distributions anymore.

Reply:
humm, for me, `rosdep` is so general that we can use non-ROS environment, and also we still have a robot with outdated Ubuntu distros (mostly because of old proprietary  device drivers for sensor/actuator I/Os. So we prefer backward compatibility.

Reply:
Using the `ubuntu` keyword as suggested will still allow you to use old ubuntu distro's I assume.

Reply:
> Using the `ubuntu` keyword as suggested will still allow you to use old ubuntu distro's I assume.

Only if the package name is the same for all distributions. In this case distros prior to xenial resolve to `python-wxgtk2.8` not `python-wxgtk3.0`.